htdocs/accountancy/journal/bankjournal.php

```diff
@@ -137,8 +145,14 @@
 $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."bank_url as bu3 ON bu3.fk_bank = b.rowid AND bu3.type='payment'";
 $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."bank_url as bu4 ON bu4.fk_bank = b.rowid AND bu4.type='payment_supplier'";
 $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe as soc on bu1.url_id=soc.rowid";
+if (empty($conf->global->SOCIETE_DISABLE_PARENTCOMPANY) && !empty($conf->global->ACCOUNTING_USE_PARENT_COMPANY)) {
+	$sql .= " JOIN ".MAIN_DB_PREFIX."societe as soc2 ON soc2.rowid = soc.parent AND soc.parent IS NOT NULL";
```

Comment:
Using a JOIN here looks strange. it means we won't be able to process record without parent company.
I think option ACCOUNTING_USE_PARENT_COMPANY means "Use account of parant company IF there is a parent company", right ?